src/data/JOBS.ts

```diff
@@ -16,6 +16,14 @@ const colors = {
 	misc: '#767676',
 }
 
+export const STATS = {
```

Comment:
This file is TS, so we can lean on the magic of `enum`s for this.

```ts
export enum Stat { // note singular
	STR,
	DEX,
	// ...
}
```

That way the job interface can include `mainStat: Stat`, which is way more restrictive.

I'm 50/50 on `STR, DEX, ...` and `STRENGTH, DEXTERITY, ...` - maybe erring towards the latter.

Reply:
Thanks for the input! I don't really use either JavaScript or TypeScript, so I didn't know if they'd play nicely exporting between the two, but it looks like it just passes over the integer value and works well.

I've updated the object literal to match the enum you have above, with the stat long names in place of their three letter abbreviations.

Any opinion on `player.mainStat` versus `player.stat` in the object passed around?

Reply:
Actually, I think they're called Attributes in game; but I'm away from a computer right now to check. Perhaps that's better.